Formula/afl-fuzz.rb

```diff
@@ -15,7 +15,7 @@ def install
     # behaving correctly!" in a nested login shell.
     # Reported to lcamtuf@coredump.cx 6th Apr 2016.
```

Comment:
If this `inreplace` is removed then `afl-fuzz` does not build, as far as I understand.